docs/source/miniconda.rst

```diff
@@ -50,7 +50,7 @@ Linux installers
    :header: Python version,Name,Size,SHA256 hash
    :widths: 5, 10, 5, 80
 
-   Python 3.9,`Miniconda3 Linux 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh>`_,58.6 MiB,``b4e46fcc8029e2cfa731b788f25b1d36``
+   Python 3.9,`Miniconda3 Linux 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh>`_,58.6 MiB,``536817d1b14cb1ada88900f5be51ce0a5e042bae178b5550e62f61e223deae7c``
```

Comment:
Yup, that's what I get when I download it.  (Sums were generated on  Zeus, so just making sure they're the same as what is fetched.)